apache/yunikorn-site
# [YUNIKORN-1386] Adding Chinese translation of deployment modes

# What is this issue for?
Translate current user_guide/deployment_modes

# Issue type
- [x]  sub-task 

# Related Apache jira link
https://issues.apache.org/jira/browse/YUNIKORN-1386
https://issues.apache.org/jira/browse/YUNIKORN-1032

# How to test
1. ./local_build run (make sure all related json files  are created in  .docusaurus)
2. ./local_build run zh-cn (This step will create additional translated json files in .docusaurus)

# screenshot
<img width="1267" alt="image" src="https://user-images.githubusercontent.com/45888688/202140586-08560b5e-df4d-4797-ad6e-dc493d43c7f9.png">
